index.html

```diff
@@ -2406,6 +2634,1425 @@ <h2>
         </p>
       </div>
     </section>
+    <section id='automation'>
+      <h2>
+        Automation
+      </h2>
+      <p>
+        For the purposes of [=user agent=] automation and application testing, this document defines extensions to the [[WebDriver-BiDi]] expecification.
+        It is OPTIONAL for a [=user agent=] to support them.
+      </p>
+      <p>
+        Each [=top-level traversable=] has a <dfn>simulated gamepad map</dfn> which is an [=ordered map=] of [=version 4 UUIDs=] to [=gamepad input sources=].
+      </p>
+      <p>
+        A <dfn>simulated gamepad</dfn> is a software-defined [=gamepad=] that behaves like a physical gamepad and has [=input controls=], [=output controls=], and a [=gamepad identifier string=].
+      </p>
+      <p>
+        A [=simulated gamepad=] is not a physical device and does not have an [=input control layout=].
+        If a [=simulated gamepad=] has device identifiers that match a [=gamepad=] with a known [=input control layout=], the [=user agent=] SHOULD process inputs on the [=simulated gamepad=] as if it had the same [=input control layout=].
+      </p>
+      <section>
+        <h3>
+          The emulation module
+        </h3>
+        <section>
+          <h4>
+            Types
+          </h4>
+          <section>
+            <h5>
+              The emulation.GamepadBounds Type
+            </h5>
+            <pre class="cddl">
+              emulation.GamepadBounds = {
+                max: float64,
+                ? min: float64,
```

Comment:
You can define `.default` in CDDL like this:
```suggestion
                ? min: float64 .default 0.0,
```